src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -101,26 +123,70 @@ static bool queryForward(ConnectionInfo* ciP, QueryContextRequest* qcrP, QueryCo
 
 
   //
-  // 2. Render the string of the request we want to forward
+  // 2. Send the request to the Context Provider (and await the reply)
+  //    Render the payload if V1 forward
   //
-  std::string  payload;
-  TIMED_RENDER(payload = qcrP->toJsonV1());
-
-  char* cleanPayload = (char*) payload.c_str();;
-
-  //
-  // 3. Send the request to the Context Provider (and await the reply)
   // FIXME P7: Should Rush be used?
   //
-  std::string     verb         = "POST";
-  std::string     resource     = prefix + "/queryContext";
+  std::string     payload;
+  std::string     verb;
+  std::string     resource;
   std::string     tenant       = ciP->tenant;
   std::string     servicePath  = (ciP->httpHeaders.servicePathReceived == true)? ciP->httpHeaders.servicePath : "";
-  std::string     mimeType     = "application/json";
+  std::string     mimeType;
+
+  LM_TMP(("qcrP->forwardingMode == '%s'", qcrP->forwardingMode.c_str()));
```

Comment:
Please check that no LM_TMP reamins in the PR before merging.

Reply:
Sorry about that.
Fixed in c522e4e